.cfignore

```diff
@@ -0,0 +1,2 @@
+docker-compose.yml
+Dockerfile
```

Comment:
The [CloudFoundry docs on .cfignore](https://docs.cloudfoundry.org/devguide/deploy-apps/prepare-to-deploy.html#exclude) mentions excluding `/tmp` and `/log` directories; are there any temporary or log directories we'd want to add here? `node_modules`?

Reply:
The Node modules are getting pushed to cloud.gov. The static files generated by the Node modules are going to be copied to S3 with the `collectstatic` command but the node modules are with the code in cloud.gov. In the future, we will build the node modules in Circle, that is when we would add it to the .cfignore. 

I am not seeing a tmp or a log directory, so I don't think we need it but no harm in adding it.

Reply:
This is helpful context! I was asking about the `/tmp` and `log` directories in an open-question way. We certainly don't need to add those lines to the .cfignore file if those directories do not exist.

Reply:
In fact — let's remove `/tmp` and `/log` from here since they don't exist.